public/data/akshatmittal61.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Akshat Mittal",
+  "type": "personal",
+  "bio": "MERN Stack Developer | Competitive Programmer",
+  "avatar": "https://github.com/akshatmittal61.png",
+  "links": [
+    {
+      "name": "Veiw my projects on GitHub",
+      "url": "https://github.com/akshatmittal61",
+      "icon": "github"
+    },
+    {
+      "name": "Mail me to talk",
+      "url": "mailto:akshatmittal2506@gmail.com",
+      "icon": "email"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/akshatmittal61",
+      "icon": "twitter"
+    },
+    {
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/akshatmittal61/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "See my posts on InG",
+      "url": "https://instagram.com/akshatmittal61",
+      "icon": "instagram"
+    },
+    {
+      "name": "Join me on your fb",
+      "url": "https://www.facebook.com/akshatmittal61/",
+      "icon": "facebook"
+    }
+  ],
+  "milestones": []
```

Comment:
```suggestion
  ]
```

Reply:
Yeah Okay, I just thought that maybe sometime in the future I would add them, so I left the empty array. I'll delete it now though.